mesonbuild/cargo/builder.py

```diff
@@ -26,200 +26,55 @@ def _token(tid: str, filename: str, value: mparser.TV_TokenTypes) -> mparser.Tok
     return mparser.Token(tid, filename, -1, -1, -1, (-1, -1), value)
```

Comment:
Just a passing by comment: the `cargo:` prefix in the commit message first line does not seem correct.